### DISCUSSION TITLE
Show Estimated Wait Times in SF

### DISCUSSION THREAD

**Discussion by sr71-yf12** [amazon-connect/amazon-connect-salesforce-scv#176] [Q&A]
Is there a standard way of showcasing the Estimated Wait Time in a Queue?

My approach was to use the Cloudwatch Monitoring Dashboard and Longest Wait time (LongestQueueWaitTime) attribute to show the number by each queue. Would love to hear other thoughts on the subject.

**Comment by dcarbonetti** [comment]
Most people use queue metrics

this blog should help:
https://aws.amazon.com/blogs/contact-center/optimize-customer-wait-experience-in-queue-using-amazon-connect/
